samples/net/sockets/socketpair/sample.yaml

```diff
@@ -15,4 +15,6 @@ tests:
     platform_exclude: s32k148_evb/s32k148
   sample.net.sockets.socketpair.s32k148_evb:
     extra_args: SHIELD=nxp_adtja1101
-    platform_allow: s32k148_evb/s32k148
+    platform_allow:
+      - s32k148_evb/s32k148
+      - qemu_x86
```

Comment:
that looks wrong. This is a specific scenaion for some board, should not have qemu_x86, which is already covered in the previous one. Just move the integration platforms like from common to the main scenario instead.

Reply:
Fixed!